pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
```

Comment:
I haven't used toml file before. Are these value referenced by something?

Reply:
This is the way to specify your project setup now and can reference other tools and requirements as well `[tool.xyz]`. For example, poetry would look different than this.
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

https://peps.python.org/pep-0517/